kolibri/content/middleware.py

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
This middleware was what allowed most of the other simplifications to occur. Now, anything that happens in the context of a request to a URL with a `channel_id` component will automatically reference the correct DB.